cmd/program/program.go

```diff
@@ -103,6 +103,7 @@ var (
 	sqliteDriver   = []string{"github.com/mattn/go-sqlite3"}
 	redisDriver    = []string{"github.com/redis/go-redis/v9"}
 	mongoDriver    = []string{"go.mongodb.org/mongo-driver"}
+	scyllaDriver   = []string{"github.com/gocql/gocql"}
```

Comment:
```suggestion
	scyllaDriver   = []string{"github.com/scylladb/gocql"}
```

Reply:
Since we have to replace the module, I managed to add a GoCQL driver and then a a replaceable right after it.

Should be clean enough for now.